ArkhamOverlay/Pages/Overlay/OverlayController.cs

```diff
@@ -16,56 +16,51 @@
 using ArkhamOverlay.Common.Services;
 using ArkhamOverlay.Common.Events;
 using ArkhamOverlay.Common.Enums;
+using ArkhamOverlay.Events;
+using ArkhamOverlay.Common.Utils;
 
 namespace ArkhamOverlay.Pages.Overlay {
     public class OverlayController : Controller<OverlayView, OverlayViewModel> {
         private readonly AppData _appData;
         private readonly LoggingService _logger;
         private readonly Configuration _configuration;
+        private readonly IEventBus _eventBus;
+        private readonly ICardZoneManager _topCardZoneManager;
+        private readonly ICardZoneManager _bottomCardZoneManager;
+        private readonly IList<ICardZoneManager> _cardZoneManagers = new List<ICardZoneManager>();
```

Comment:
Not quire sure what is the point of this object. Is it meant to accommodate multiple CardZones in the future?

Reply:
There are two areas for displaying card zone- one on the top, one on the bottom. Each of these managers (right now there are two) manages the visual cards for its zone.